pkg/apis/fleet.cattle.io/v1alpha1/bundle.go

```diff
@@ -346,3 +350,9 @@ type Content struct {
 
 	Content []byte `json:"content,omitempty"`
 }
+
+type ResyncPolicy struct {
+	MaxRetries   int    `json:"maxRetries,omitempty"`
+	ResyncDelay  string `json:"resyncDelay,omitempty"`
```

Comment:
Can you make these two fields as `*metav1.Duration` so it can be validated and parsed at api levels? Then you don't have to deal with it in controller code.

Reply:
done. Thanks for the pointer.